odysseus/convertFtbQuests.ts

```diff
@@ -275,6 +275,15 @@ function convertItemId(id: ResourceLocation): ResourceLocation {
     return id == 'ftbquests:book' ? 'heracles:quest_book' : id
 }
 
+function convertItemNbt(nbt: JsonObject | undefined): JsonObject | undefined {
+    if (nbt === undefined) return undefined;
+    let outNbt = nbt;
```

Comment:
This is not a copy. You're modifying both outNbt and nbt.

Reply:
what's the most appropriate way to copy?

Reply:
I'd probably do `{... nbt}`.

Reply:
done in 4b23975